src/components/Search/index.tsx

```diff
@@ -96,23 +92,28 @@ function Search({query, policyIDs, sortBy, sortOrder}: SearchProps) {
     const isLoadingMoreItems = !isLoadingItems && searchResults?.search?.isLoading && searchResults?.search?.offset > 0;
     const shouldShowEmptyState = !isLoadingItems && isEmptyObject(searchResults?.data);
 
-    if (isLoadingItems) {
-        return <TableListItemSkeleton shouldAnimate />;
-    }
-
-    if (shouldShowEmptyState) {
-        return <EmptySearchView />;
+    if (isLoadingItems || shouldShowEmptyState) {
+        return (
+            <>
+                <SearchPageHeader
+                    selectedItems={{}}
+                    query={query}
+                    hash={hash}
+                />
+                {isLoadingItems ? <TableListItemSkeleton shouldAnimate /> : <EmptySearchView />}
+            </>
+        );
```

Comment:
Previously we had 2 simple ifs
Now we have 1 if with OR expression and then a ternary inside.

I would prefer simpler previous logic even if it means repeated `<SearchPageHeader>`.